### PR TITLE
Add note about 1.5.1's --extra-packages flag

### PR DESCRIPTION
**Pull request by merlinpatt**
One other thing that might be worth adding is to let people know to remove the `force-ssl` package. Otherwise the app keeps redirecting to https, which localhost doesn't seem to allow?

### CODE REVIEW

File: source/packages/bundle-visualizer.md [hunk 1]
```diff
@@ -47,6 +47,13 @@ $ meteor add bundle-visualizer
 $ meteor --production
 ```
 
+If you have Meteor 1.5.1, you can use the `--extra-packages` flag, which will only add the visualizer package to your app temporily.
```
Comment (klaussner) [inline source/packages/bundle-visualizer.md]: "temporarily"

**Comment by merlinpatt** [comment]
@klaussner fixed. What are your thoughts on adding the note about temporarily removing the `force-ssl` package?

**Comment by klaussner** [comment]
@merlinpatt I think the `force-ssl` package doesn't (or shouldn't?) redirect to `https://` on localhost.

**Comment by merlinpatt** [comment]
@klaussner if you're running it regularly, it doesn't. But if you use the `--production` flag, which the visualizer requires, it does.

**Comment by klaussner** [comment]
Hm, I just tried running an app with `meteor run --production --extra-packages bundle-visualizer` (and `force-ssl@1.0.14`) and `http://localhost:3000` wasn't redirected. `force-ssl` [checks if the server is running locally](https://github.com/meteor/meteor/blob/b52c6587d7542c0f27481a3bee8c65be06068ac1/packages/force-ssl/force_ssl_server.js#L23), so it shouldn't redirect on localhost, even if it's a production build.

**Comment by merlinpatt** [comment]
Okay. I'll try it again. Maybe it was a bug on my end. I'll file a bug report if so. So I believe this PR is good to go

**Comment by abernix** [comment]
Since we have versioned docs though I just added `1.5.1` and I made a few modifications so as not to make "if you have 1.5.1" comments, but still kept your commit.  Thanks! 😸

**Pull request merged**